placeSpecificContent/hq/chatterboxes/README.md

```diff
@@ -0,0 +1,26 @@
+# CHATTERBOX
```

Comment:
Please define somewhere in here the necessary parent/child relationships of the various components.

Reply:
done